src/css/custom.css

```diff
@@ -248,6 +248,10 @@ h1.docTitle_node_modules-\@docusaurus-theme-classic-src-theme-DocItem- {
     mask: url(/img/team.svg) no-repeat 100% 100%;
     mask-size: cover;
 }
+.footer__faq:before {
+    mask: url(/img/faq.svg) no-repeat 100% 100%;
```

Comment:
Why was this added to the footer css?

Reply:
It's the svg icon to the footer FAQ link.
If FAQ link in the footer is suggested to remove these CSS changes will be discarded too.